### PR TITLE
feat(layout): add profile containers

### PR DESCRIPTION
**Pull request by artyorsh**
### Please read and mark the following check list before creating a pull request:

 - [x] I read and followed the [CONTRIBUTING.md](https://github.com/akveo/kittenTricks/blob/master/CONTRIBUTING.md) guide.

 #### Short description of what this resolves:

* Integrates profile containers (2x)
* Refactors common `profile` components names
* Fixes `ReactionButton` paddings

### CODE REVIEW

File: src/components/social/profile/profileActivityList2Item.component.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  ThemeType,
+  withStyles,
+} from '@kitten/theme';
+
+interface ListDerivedProps {
+  index?: number;
+}
+
+// @ts-ignore
+interface TouchableProps extends TouchableOpacityProps {
```
Comment (32penkin) [inline src/components/social/profile/profileActivityList2Item.component.tsx:20]: Move to separate place as used in not one place.

File: src/components/social/profile/profileActivityList2Item.component.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  onPress: (index: number) => void;
+}
+
+interface ComponentProps extends ListDerivedProps, TouchableProps {
+  source: ImageSourcePropType;
+}
+
+export type ProfileActivityList2ItemProps = ThemedComponentProps & ComponentProps & ComponentProps;
```
Comment (32penkin) [inline src/components/social/profile/profileActivityList2Item.component.tsx]: ```ThemedComponentProps & ComponentProps & TouchableProps```?

File: src/components/social/profile/profileActivityList2Item.component.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+          style={themedStyle.image}
+          source={source}
+        />
+      </TouchableOpacity>
+    );
+  }
+}
+
+export const ProfileActivityList2Item = withStyles(ProfileActivityList2ItemComponent, (theme: ThemeType) => ({
+  container: {},
```
Comment (32penkin) [inline src/components/social/profile/profileActivityList2Item.component.tsx]: Suggest remove empty objects. This is not a framework

File: src/components/social/profile/profileFriendList.component.tsx [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+        style={themedStyle.item}
+        activeOpacity={0.75}
+        photo={{ uri: item.photo }}
+        name={item.firstName}
+        onPress={this.onItemPress}
+      />
+    );
+  };
+
+  private renderItem = (info: ListRenderItemInfo<ProfileModel>): ListItemElement => {
```
Comment (32penkin) [inline src/components/social/profile/profileFriendList.component.tsx:53]: Looks like useless method in almost all lists. You can pass index like a component property.

File: src/components/social/profile/profileFriendListItem.component.tsx [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+} from '@kitten/theme';
+import { Avatar } from '@kitten/ui';
+import { Text } from '@src/components/common';
+
+interface ListDerivedProps {
+  index?: number;
+}
+
+// @ts-ignore
+interface TouchableProps extends TouchableOpacityProps {
```
Comment (32penkin) [inline src/components/social/profile/profileFriendListItem.component.tsx:20]: Separate place

File: src/components/social/profile/profileFriendListItem.component.tsx [hunk 6]
```diff
@@ -0,0 +1,71 @@
+import React from 'react';
```
Comment (32penkin) [inline src/components/social/profile/profileFriendListItem.component.tsx:1]: Why don't you wanna implement this component in ProfileFriendsList if it most likely will not be reused? In this case you have to import a lot of dependencies even in each 'small' component. I guess this is not good.

File: src/containers/layouts/social/profile7/profile7.component.tsx [hunk 7]
```diff
@@ -0,0 +1,207 @@
...
+  profileSectionContent: {
+    marginTop: 8,
+  },
+  socialsLabel: {
+    color: theme['color-white'],
+  },
+  followButton: {
+    flex: 1,
+    marginRight: 4,
+    textTransform: 'uppercase',
```
Comment (32penkin) [inline src/containers/layouts/social/profile7/profile7.component.tsx:182]: I guess string value in button is static. Just write 'FOLLOW' and don't create extra fields in the objects.

File: src/containers/layouts/social/profile7/profile7.component.tsx [hunk 8]
```diff
@@ -0,0 +1,207 @@
...
+  },
+  messageButton: {
+    flex: 1,
+    marginLeft: 4,
+    textTransform: 'uppercase',
```
Comment (32penkin) [inline src/containers/layouts/social/profile7/profile7.component.tsx:187]: Useless

File: src/core/data/profile.ts [hunk 9]
```diff
@@ -53,27 +56,163 @@ export const profile3: Profile = {
...
+      source: `https://images.unsplash.com/photo-1473768961734-a7222f539688`,
+    },
+    {
+      category: 'style',
+      source: `https://images.unsplash.com/photo-1471560090527-d1af5e4e6eb6`,
+    },
+  ]
+;
+
+export const categorisedProfileActivity1: CategorisedProfileActivity = {
```
Comment (32penkin) [inline src/core/data/profile.ts:155]: Can it be implemented through 'filter' method. Looks like codegeneration

**Review by 32penkin — APPROVED** [review]

File: src/components/social/profile/profileFriendList.component.tsx (hunk 4 above)
Comment (artyorsh) [reply src/components/social/profile/profileFriendList.component.tsx:53]: This should be provided by framework, but still needs discussion
Same behavior could be achieved when wrapping list item view into `ListItem` component

File: src/core/data/profile.ts (hunk 9 above)
Comment (artyorsh) [reply src/core/data/profile.ts:155]: I guess we don't need implementing any logic on data, because it's just a demo app

**Pull request merged**